lighthouse-core/audits/metrics/first-contentful-paint.js

```diff
@@ -54,12 +61,13 @@ class FirstContentfulPaint extends Audit {
     const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
     const metricComputationData = {trace, devtoolsLog, settings: context.settings};
     const metricResult = await ComputedFcp.request(metricComputationData, context);
+    const scoreOptions = context.options[artifacts.TestedAsMobileDevice ? 'mobile' : 'desktop'];
 
     return {
       score: Audit.computeLogNormalScore(
         metricResult.timing,
-        context.options.scorePODR,
-        context.options.scoreMedian
+        scoreOptions.scorePODR,
```

Comment:
@patrickhulce @paulirish @brendankenny Good?

Reply:
probably should still note this as a breaking change in release notes, but this WFM 👍

Reply:
agreed. we should adopt a way to signify breaking changes for release-notes-writing time.